packages/cli/src/cli/start.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2025 Bytedance, Inc. and its affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GUIAgent } from '@ui-tars/sdk';
+import inquirer from 'inquirer';
```

Comment:
`inquirer ` is good but its ui is a bit outdated, do you consider https://github.com/bombshell-dev/clack to replace it